src/oscar/apps/offer/abstract_models.py

```diff
@@ -625,8 +627,16 @@ def round(self, amount):
         """
         Apply rounding to discount amount
         """
-        if hasattr(settings, 'OSCAR_OFFER_ROUNDING_FUNCTION'):
-            return settings.OSCAR_OFFER_ROUNDING_FUNCTION(amount)
+        rounding_function = getattr(settings, 'OSCAR_OFFER_ROUNDING_FUNCTION', None)
+        if callable(rounding_function):
+            return rounding_function(amount)
+        elif isinstance(rounding_function, six.string_types):
+            try:
+                rounding_function = import_string(rounding_function)
```

Comment:
Think we should consider doing this once and wrapping it in an `lru_cache`, the way we do here: https://github.com/django-oscar/django-oscar/blob/master/src/oscar/core/loading.py#L34-L36

Reference: https://wiki.python.org/moin/PythonSpeed/PerformanceTips#Import_Statement_Overhead